Receipt/Sources/ListReceiptController.swift

```diff
@@ -39,12 +39,15 @@ public final class ListReceiptController: UITableViewController {
         title = "title_receipts".localized()
         largeTitle()
         setViewBackgroundColor()
-
         navigationItem.backBarButtonItem = UIBarButtonItem.back
+        initializePresenter()
+        setupListReceiptTableView()
+    }
+
+    private func initializePresenter() {
         presenter = ListReceiptPresenter(view: self,
                                          prepaidCardToken: initializationData?[InitializationDataField.prepaidCardToken]
                                             as? String)
-        setupListReceiptTableView()
         presenter.listReceipts()
```

Comment:
presenter.listReceipts() should move back to viewDidLoad. Initialize means just to initialize the presenter and not perform any action in there. Same goes for other Controllers as well

Reply:
modified initializePresenter with only initializing presenter.